testclient/mainwindow.cpp

```diff
@@ -236,7 +236,7 @@ void MainWindow::mvAxisPBClicked()
 void MainWindow::fanSpeedPBClicked()
 {
     addSLog(GCode::toString(GCode::M106));
-    core->pushCommand(GCode::toCommand(GCode::M106, ui->fanSpeedSelCB->currentText().at(4), ui->fanSpeedSB->cleanText()));
+    core->setFanSpeed(ui->fanSpeedSB->value(), QString(ui->fanSpeedSelCB->currentText().at(4)).toInt());
```

Comment:
We seam to need to have to use QString to convert the pulled QChar from the QComboBox to an int.